templates/types/streaming/nextjs/app/components/ui/chat/markdown.tsx

```diff
@@ -28,8 +29,13 @@ const preprocessLaTeX = (content: string) => {
   return inlineProcessedContent;
 };
 
+const preprocessUrl = (content: string) => {
+  return replaceAttachmentUrl(content);
```

Comment:
this is showing the inline images in a message see?

![image](https://github.com/run-llama/create-llama/assets/17126/30393b1e-9363-4604-81de-4a8ffb754f7f)

Reply:
yes, but before and after the widget there might be some markdown text from the LLM

how about:
1. if we see an attachment in the markdown, we replace it with rendering an image widget
2. this image widget shows the image AND allows us to do something with the file (e.g. download it - the functionality of the current interpreter widget)

but for the start (this PR) we can just show the image inline and not use the interpreter widget at all (keep the example simple!)